packages/lab/stories/tabs-next/tabs-next.stories.tsx

```diff
@@ -203,18 +203,31 @@ WithBadge.args = {
 };
 
 export const Overflow: StoryFn<typeof TabsNext> = (args) => {
+  const [width, setWidth] = useState(400);
```

Comment:
Is this actually needed, it should work with the resize in the toolbar?

Reply:
Easiest for a test i guess